practitioner-service/src/main/resources/bootstrap-local.yml

```diff
@@ -2,3 +2,5 @@ spring:
   cloud:
     config:
       enabled: false
+      discovery:
+        enabled: false
```

Comment:
if cloud.config.enabled=false anyway, then is it required to also set the other more inner properties?

Reply:
Removed